src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -59,37 +53,21 @@ const SetAclPermissionForm = () => {
     try {
       await setDocAclPermission(session, docType, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for ${docType}.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
         `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
```

Comment:
I think the "to" should also be a ternary based on `permissions.read`.  I'd prefer it to be `${permissions.read ? 'to' : 'from'}` as just "to" doesn't really work for `Revoked permissions to {person} for {doctype}`.

Reply:
![Screen Shot 2023-08-22 at 7 57 34 AM](https://github.com/codeforpdx/PASS/assets/81979567/c249c4cd-5283-413b-8106-81d26bfa91ae)
Like this?